op-supervisor/supervisor/backend/db/logs/db.go

```diff
@@ -276,7 +301,7 @@ func (db *DB) newIteratorAt(blockNum uint64, logIndex uint32) (*iterator, error)
 	if err != nil {
```

Comment:
This condition is never true because if `err != nil` then one of the checks on 292 or 295 will return.